archive/generic-core-terms.md

```diff
@@ -0,0 +1,46 @@
+
+# Expectations
+
+When working with us according to this policy, you can expect us to:
+
+- Work with you to understand and validate your report, including a timely initial response to the submission;
+- Work to remediate discovered vulnerabilities in a timely manner; and
+- Recognize your contribution to improving our security if you are the first to report a unique vulnerability, and your report triggers a code or configuration change.
+
+# Scope
+
+# Out-of-Scope
+
+# Rewards
+
+# Disclosure Policy
+
+# Ground Rules
+
+To encourage vulnerability research and to avoid any confusion between legitimate research and malicious attack, we ask that you attempt, in good faith, to:
+
+- Play by the rules. This includes following this policy any other relevant agreements;
+- Report any vulnerability you’ve discovered promptly;
+- Avoid violating the privacy of others, disrupting our systems, destroying data, and/or harming user experience;
+- Use only the Official Channels to discuss vulnerability information with us;
+- Handle the confidentiality of details of any discovered vulnerabilities according to our Disclosure Policy;
+- Perform testing only on in-scope systems, and respect systems and activities which are out-of-scope;
+- If a vulnerability provides unintended access to data: Limit the amount of data you access to the minimum required for effectively demonstrating a Proof of Concept; and cease testing and submit a report immediately if you encounter any user data during testing, such as Personally Identifiable Information (PII), Personal Healthcare Information (PHI), credit card data, or proprietary information;
+- You should only interact with test accounts you own or with explicit permission from the account holder; and
+- Do not engage in extortion.
+
+# Safe Harbor
+
+When conducting vulnerability research according to this policy, we consider this research conducted under this policy to be:
+
+- Authorized in view of any applicable anti-hacking laws, and we will not initiate or support legal action against you for accidental, good faith violations of this policy;
+- Authorized in view of relevant anti-circumvention laws, and we will not bring a claim against you for circumvention of technology controls;
+- Exempt from restrictions in our Acceptable Usage Policy that would interfere with conducting security research, and we waive those restrictions on a limited basis; and
+- Lawful, helpful to the overall security of the Internet, and conducted in good faith.
+
+You are expected, as always, to comply with all applicable laws. If legal action is initiated by a third party against you and you have complied with this policy, we will take steps to make it known that your actions were conducted in compliance with this policy. 
```

Comment:
Suggested insertion: 

Please note that the safe harbor applies only to legal claims under the control of the organization participating in this policy, and that the policy does not bind independent third parties.